.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,44 @@
+name: pipeline
+
+on:
+  pull_request:
+    branches:
+      - master
+  schedule:
+      - cron:  '30 5,17 * * *'
+
+jobs:
+  pipeline:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node: [14]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Setup node env
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Cache node_modules
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run code style check
+        run: npm run lint  
+        
+      - name: Run Svelte check
+        continue-on-error: true
```

Comment:
What's the use of `continue-on-error` here? We want to fail on error, and it's the last step anyway so it doesn't even make sense.